t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -276,6 +276,31 @@ func (s *WhisperMailboxSuite) TestRequestMessagesInGroupChat() {
 	s.Require().Equal(helloWorldMessage, messages[0]["payload"].(string))
 }
 
+func (s *WhisperMailboxSuite) TestSendMessageWithoutSubscription() {
+	aliceBackend, stop := s.startBackend("alice")
+	defer stop()
+
+	// we need to wain >= whisper.DefaultSyncAllowance seconds to update
```

Comment:
s/wain/wait/

Any other way to signal update is done or timeout has happened?

Reply:
Fixed.